packages/Symfony/src/Rector/Identical/ResponseStatusCodeRector.php

```diff
@@ -0,0 +1,133 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Identical;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\BinaryOp\Identical;
+use PhpParser\NodeDumper;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+final class ResponseStatusCodeRector extends AbstractRector
+{
+    /**
+     * @var array
+     */
+    private $codeToConst = [];
+
+    public function __construct()
+    {
+        $ref = new \ReflectionClass('Symfony\\Component\\HttpFoundation\\Response');
+
+        foreach ($ref->getConstants() as $key => $value) {
+            if (strpos($key, 'HTTP_') === 0) {
+                $this->codeToConst[$value] = $key;
+            }
+        }
+    }
+
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition('Turns status code numbers to constants', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+class SomeController
+{
+    public function index()
+    {
+        $response = new \Symfony\Component\HttpFoundation\Response();
+        $response->setStatusCode(200);
+        
+        if ($response->getStatusCode() === 200) {}
+    }
+}
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+class SomeController
+{
+    public function index()
+    {
+        $response = new \Symfony\Component\HttpFoundation\Response();
+        $response->setStatusCode(\Symfony\Component\HttpFoundation\Response::HTTP_OK);
+        
+        if ($response->getStatusCode() === \Symfony\Component\HttpFoundation\Response::HTTP_OK) {}
+    }
+}
+CODE_SAMPLE
+
+            )
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [Identical::class, Node\Expr\MethodCall::class];
+    }
+
+    /**
+     * @param Identical|Node\Expr\MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if ($node instanceof Node\Expr\MethodCall) {
+            return $this->processMethodCall($node);
+        }
+
+        if ($node instanceof Identical) {
+            return $this->processIdentical($node);
+        }
+
+
+        return $node;
+    }
+
+    private function processMethodCall(Node\Expr\MethodCall $methodCall): ?Node\Expr\MethodCall
+    {
+        if (!$this->isType($methodCall->var, 'Symfony\\Component\\HttpFoundation\\Response') || $this->getName($methodCall)!== 'setStatusCode') {
+            return null;
+        }
+
+        $statusCode = $methodCall->args[0]->value;
+
+        if (! $statusCode instanceof Node\Scalar\LNumber) {
+            return null;
+        }
+
+        if (!isset($this->codeToConst[$statusCode->value])) {
+            return null;
+        }
+
+        $methodCall->args[0] = new Node\Arg(new Node\Expr\ClassConstFetch(new Node\Name('\\Symfony\\Component\\HttpFoundation\\Response'), new Node\Identifier($this->codeToConst[$statusCode->value])));
```

Comment:
Hey @TomasVotruba, is there a way to add a use statement?

Reply:
Hi, some people preffer FQN, some import it. So that's job of coding standard. 

Also, you can use `$this->createClassConstant()` methodh here.